compiler/compile/OMRCompilation.cpp

```diff
@@ -978,6 +979,9 @@ int32_t OMR::Compilation::compile()
       methodInfo->setProfileInfo(NULL);
 #endif
 
+   // Create the compile time profiler
+   TR::CompileTimeProfiler perf("compileTimePerf");
```

Comment:
This would have to be guarded with an `#if defined(LINUX)` as well; either that or the implementation of `TR::CompileTimeProfiler` would need to moved to `compiler/x/env/CompileTimeProfiler.hpp` and an empty implementation would have to be added to `compiler/env/CompileTimeProfiler.hpp`.

Reply:
Should this be guarded in some way to save creating it if it won't be used?

Reply:
The implementation followed that of `RegionProfiler`, making use of RAII to control the duration of the profile and perform a runtime check internally. This has been modified to avoid the overhead in the disabled case.